### PR TITLE
Using the new biscuit tool to merge CpG counts

### PR DESCRIPTION
**Pull request by chrisamiller**
removing my homebrew script, as it turns out there's a utility built in that I missed!

### CODE REVIEW

File: bsvcf2bed [hunk 1]
```diff
@@ -5,13 +5,14 @@ set -o errexit
...
-#arg3 = /path/to/output/cpgs.bedgraph
+#arg2 = /path/to/reference.fa
+#arg3 = /path/to/output/cpgs.bed.gz
+#arg4 = /path/to/output/cpgs.bedgraph
 
 #Creates a gzipped bed and a bedgraph that leaves out MT, random, GL contigs, etc
-if [[ "$#" -lt 3 ]]; then
-    echo "not enough parameters specified. Expected biscuit_vcf, output_cpgs.bed.gz, output_cpgs.bedgrap"
+if [[ "$#" -lt 4 ]]; then
+    echo "not enough parameters specified. Expected biscuit_vcf, reference_fasta, output_cpgs.bed.gz, output_cpgs.bedgrap"
```
Comment (tmooney) [inline bsvcf2bed]: There's an `h` missing from the end of this line.

File: bsvcf2bed [hunk 2]
```diff
@@ -5,13 +5,14 @@ set -o errexit
...
     exit 1
 fi
 
-perl /usr/bin/biscuitVcfToBed.pl "$1" |  tee >(/bin/gzip >"$2") | cut -f 1-4 | sort -k1,1 -k2,2n -S 12G | /usr/bin/perl -ne 'print $_ if $_ =~ /^(chr)?[1-9]?[0-9|X|Y]\s/' >"$3"
+/usr/bin/biscuit vcf2bed -t cg -k 1 -e "$1" | /usr/bin/biscuit mergecg "$2.gz" /dev/stdin -k 2 |  tee >(/bin/gzip >"$3") | cut -f 1-4 | sort -k1,1 -k2,2n -S 12G | /usr/bin/perl -ne 'print $_ if $_ =~ /^(chr)?[1-9]?[0-9|X|Y]\s/' >"$4"
```
Comment (tmooney) [inline bsvcf2bed]: I kind of wish we could make the `.gz` a secondaryFile for the reference for this pipeline... but that doesn't really work with the `string` version.  It's a bit of an interesting that that this script expects to be passed most of a filename for `$2`.

**Review by tmooney — COMMENTED** [review]
I'm a little sad; I was happy to get to look at some Perl code again for #3!  😄

File: bsvcf2bed (hunk 2 above)
Comment (chrisamiller) [reply bsvcf2bed]: I just figured that it was easier to pass in the same reference fasta as everything else in the pipeline.  It's pretty weird that it only accepts the gzipped one, but in my testing, that's the only way I could get it to work.

Comment (chrisamiller) [reply bsvcf2bed]: Added an error with a little more info

Comment (chrisamiller) [reply bsvcf2bed]: removed this completely - the reference file was jacked up, which caused the error.  the whole gz thing is unnecessary

File: bsvcf2bed [hunk 3]
```diff
@@ -1,17 +1,18 @@
-#!/bin/bash
+bsvcf2bed#!/bin/bash
```
Comment (tmooney) [inline bsvcf2bed]: Looks like this `bsvcf2bed` snuck in here.
```suggestion
#!/bin/bash
```

**Review by tmooney — COMMENTED** [review]
Cool, I'm glad we could remove that `.gz` business!  +1 after fixing the `#!` line 😄

**Pull request merged**